html-aam/roles-contextual.html

```diff
@@ -22,11 +22,30 @@
 <a data-testname="el-a-no-href" class="ex-generic">x</a>
 
 <!-- el-aside -->
+<aside data-testname="el-aside" data-expectedrole="complementary" class="ex">x</aside>
+<main>
+  <aside data-testname="el-aside-in-main" data-expectedrole="complementary" class="ex">x</aside>
+  <section>
+    <aside data-testname="el-aside-in-section-in-main" class="ex-generic">x</aside>
```

Comment:
noting that while this is what the spec currently says to do, because the section is without a name it is treated no differently than a `div` in the accTree.  

but if that section were given a name, then it would be a role=region, and then it would would be more appropriate for this to be generic.

https://github.com/w3c/html-aam/pull/484/files will eventually address this, but mentioning it here if we want to do anything about it now / modify html aam to at least rectify this single case

Reply:
Would it make sense to name the `<sections>` in this WPT change and to ignore the un-named `<section>` case until https://github.com/w3c/html-aam/pull/484 is merged? That way, browsers that are following to the spec word-for-word aren't dinged yet, but browsers that eagerly implement the behavior in that PR can still pass the WPT, and aren't told that:

```
<section>
    <aside data-expectedrole="generic"></aside>
</section> 
```

needs to pass.

We could also include the un-named `<section>` case in this WPT, but I'm not sure if people feel weird about testing something that isn't in the spec yet.

Reply:
If there is general agreement to the spec change and no objection, I think it's fine to merge the test. 

ARIA WG Chairs changed the process so it's now the expectation that any spec change PR will be approved, but not merged until there are 1. tests, and 2. multiple passing implementations. So if we didn't merge the test, it would cause a circular dependency with the spec PR. 😉

Reply:
I don't know if you two have fully resolved this thread, but the rest of the PR looks good to me, so I'm approving.